active/tab/tabcontrol.md

```diff
@@ -0,0 +1,580 @@
+* Modern browsers support selecting tabs when pressing CTRL and a number. Consider handling Ctrl+1 through 8, which should select that number Tab. Additionally, you can consider handling Ctrl+9 which selects the last tab (regardless of the number of tabs).
+* If users can perform more commands on a Tab than just closing the Tab (for example, pinning a Tab or duplicating a Tab), consider adding a context menu to the TabViewItem. 
+
+# API Notes
+<!-- Give a one or two line description of each API (type
+and member), or at least the ones that aren't obvious
+from their name.  These descriptions are what show up
+in IntelliSense. -->
+
+### TabView properties, events, methods
+
+| Property | Description |
+|:-------- |:----------- |
+| AddTabButtonCommand | Gets or sets the command to invoke when the Add button is tapped. |
+| AddTabButtonCommandParameter | Gets or sets the parameter to pass to the command for the Add button. |
+| CanDragTabs | Gets or sets a value that indicates whether tabs in the collection can be dragged. Default is true. |
+| CanReorderTabs | Gets or sets a value that indicates whether tabs in the collection can be reordered through user interaction. Default is true. |
+| IsAddTabButtonVisible | Determines if the plus button appears to the right of the tab strip |
+| TabItemsSource | Gets or sets an object source used to generate the tabs. |
+| TabItems | Gets the collection used to generate the tabs. |
+| ItemTemplate | Gets or sets the DataTemplate used to display each item. |
+| ItemTemplateSelector | Gets or sets a reference to a custom DataTemplateSelector logic class. The DataTemplateSelector referenced by this property returns a template to apply to items. |
+| SelectedIndex | Gets or sets the index of the selected item. |
+| SelectedItem | Gets or sets the selected item. |
+| TabStripHeader | Content to the left of the tab strip. |
+| TabStripFooter | Content to the right of the tab strip. |
+| TabViewItemWidthMode | Specifies how the tabs should be sized. Values are {Actual, Equal}. Default is Actual. |
+
+| Event | Description |
+|---|---|
+| AddTabButtonClick | Fires when the + button is clicked. |
+| SelectionChanged | Occurs when the currently selected tab changes. |
+| TabClosing | Fires when a tab is about to be closed. Can be cancelled to prevent closure. |
+| TabDraggedOutside | Fires when a Tab is dragged outside of the Tab bar. |
+
+| Method | Description |
+|---|---|
+| DependencyObject ContainerFromItem(object item) | Returns the generated container for the specified item. |
+| DependencyObject ContainerFromIndex(Int32 index) | Returns the generated container for the specified index. |
+
+### TabViewItem properties and events
+
+| Property | Description |
+|:-------- |:----------- |
+| Content | The main content that appears in the tab. |
+| Header | The content that appears inside the tab itself.  |
```

Comment:
The descriptions for `Content` and `Header` are nearly identical.

Reply:
Agreed, and I'm open to suggestions. I wasn't sure how to succinctly say "the stuff in the Header goes into the literal UI of the tab whereas the content goes into the body below the tab".

Reply:
How about something like

| Property | Description |
|:-|:-|
| Content | The content that appears in the TabView's content area when the TabViewItem is selected. |
| Header | The content that appears in the TabView's tab row to represent the tab. |